src/proxy/inbound.rs

```diff
@@ -75,12 +76,12 @@ impl Inbound {
                     cert_manager: self.cert_manager.clone(),
                 },
             };
-            let mut listener = hyper::server::conn::AddrIncoming::from_listener(self.listener)
-                .expect("hbone bind");
-            listener.set_nodelay(true);
+            let mut incoming_listener =
+                hyper::server::conn::AddrIncoming::bind(&addr).expect("hbone bind");
```

Comment:
This is changing the semantic. Its now actually binding

Reply:
Yeah, it's really bind here.